course-info.yaml

```diff
@@ -2,7 +2,7 @@ type: marketplace
 title: Introduction to Python
 language: English
 summary: |-
-  <p>This is an introductory Python course by JetBrains Academy. The presented materials are based on the official <a href="https://docs.python.org/3/">Python Documentation</a>. The target audience are the people with little or no experience in programming who would like to start learning Python.</p>
+  <p>This is an introductory Python course by JetBrains Academy. Python is the leading language used for data analysis, AI, ML, automation, and more. The presented materials are based on the official <a href="https://docs.python.org/3/">Python Documentation</a>. The target audience are the people with little or no experience in programming who would like to start learning Python.</p>
```

Comment:
<p>This is an introductory Python course by JetBrains Academy. Python is a leading language for data analysis, AI, ML, automation, and more. The course materials are based on the official <a href="https://docs.python.org/3/">Python Documentation</a>. It is designed for individuals with little to no programming experience who want to start learning Python.</p>